### ISSUE TITLE
Remove tests from conda package

### ISSUE THREAD

**Issue by ariel215** [pennmem/ram_utils#246]
Since we started including the test data in the conda package, the package size has gone from 200 KB to 90 MB. This is a lot of data  that is not, strictly speaking, necessary for release.

**Comment by mivade** [comment]
Yes, that shouldn't be included in the conda package. The trivial solution is to move tests out of the `ramutils` package in the first place.

**Comment by leondavis1** [comment]
Goal: tests should not be included in conda package, but we should still be able to automatically test against the packaged module
